drivers/serial/uart_numaker.c

```diff
@@ -0,0 +1,451 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ *  Copyright (c) 2023 Nuvoton Technology Corporation.
+ */
+
+#include <zephyr/drivers/uart.h>
+#include <zephyr/drivers/reset.h>
+#include <string.h>
+#include <zephyr/irq.h>
+#include <zephyr/logging/log.h>
+#include <NuMicro.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/clock_control_numaker.h>
+#include <zephyr/drivers/pinctrl.h>
```

Comment:
needs better sorting: libc, zephyr, HAL (check others as well)

Reply:
Applied to 5 drivers